packages/blade/src/components/Input/DropdownInputTriggers/AutoComplete.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
```

Comment:
Should we keep autocomplete outside the dropdown component? like `components/AutoComplete`? Since it is used in Dropdown as well as Bottomsheet?

Reply:
so Dropdown as a whole is used inside BottomSheet actually. So it isn't much different from SelectInput. 

https://blade.razorpay.com/?path=/docs/components-bottomsheet--with-dropdown-single-select